Bulldozer/Utility/CachedTypes.cs

```diff
@@ -110,12 +112,15 @@ public static partial class CachedTypes
 
         public static int GroupTypeMeetingLocationId = DefinedValueCache.Get( Rock.SystemGuid.DefinedValue.GROUP_LOCATION_TYPE_MEETING_LOCATION.AsGuid() ).Id;
         public static int GroupTypeCheckinTemplateId = DefinedValueCache.Get( Rock.SystemGuid.DefinedValue.GROUPTYPE_PURPOSE_CHECKIN_TEMPLATE.AsGuid() ).Id;
+        public static int GroupTypePurposeServingAreaId = DefinedValueCache.Get( "36A554CE-7815-41B9-A435-93F3D52A2828".AsGuid() ).Id;      // Serving Area defined value for group type purpose
         public static int DeviceTypeCheckinKioskId = DefinedValueCache.Get( Rock.SystemGuid.DefinedValue.DEVICE_TYPE_CHECKIN_KIOSK.AsGuid() ).Id;
 
         public static int BenevolenceApprovedStatusId = DefinedValueCache.Get( Rock.SystemGuid.DefinedValue.BENEVOLENCE_APPROVED.AsGuid() ).Id;
         public static int BenevolenceDeniedStatusId = DefinedValueCache.Get( Rock.SystemGuid.DefinedValue.BENEVOLENCE_DENIED.AsGuid() ).Id;
         public static int BenevolencePendingStatusId = DefinedValueCache.Get( Rock.SystemGuid.DefinedValue.BENEVOLENCE_PENDING.AsGuid() ).Id;
 
+        public static int FinancialAccountTypeDefindedTypeId = DefinedTypeCache.Get( Rock.SystemGuid.DefinedType.FINANCIAL_ACCOUNT_TYPE.AsGuid() ).Id;
```

Comment:
Please fix Typo `Definded`
```suggestion
        public static int FinancialAccountTypeDefinedTypeId = DefinedTypeCache.Get( Rock.SystemGuid.DefinedType.FINANCIAL_ACCOUNT_TYPE.AsGuid() ).Id;
```